### DISCUSSION TITLE
New supported device: MT09R NU1 Indoor Ceiling Cassette + MU4R25 U21 Outdoor Device

### DISCUSSION THREAD

**Discussion by mptr8956** [JanM321/esphome-lg-controller#47] [General]
Installed the first controller today, works flawlessly with the indoor device. 

Thank you very much, awesome project! :)

**Comment by JanM321** [comment]
Thanks! I've added the unit to the README.

Most people are using this controller with wall mounted indoor units so it's always good to hear it works with other types too.
